{{cookiecutter.repository_name}}/guild.yml

```diff
@@ -45,10 +45,18 @@
       flags-import: all
       output-scalars: off
       requires:
+        - sourcecode-symlink
         - notebooks
         - model
         - prepare
         - splits
+    commit:
```

Comment:
Maybe we want to call it something more specific since it only pertains to notebooks? Like `commit-notebooks`

Reply:
We discussed non-interactive notebooks in slack and I think some of this will be removed as a result